controllers/reinjection/controller.go

```diff
@@ -0,0 +1,118 @@
+package reinjection
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	chaosmeshapi "github.com/chaos-mesh/chaos-mesh/api/v1alpha1"
+	log "github.com/sirupsen/logrus"
```

Comment:
Do not introduce a new logging package, please refer to https://github.com/chaos-mesh/chaos-mesh/issues/2752 and `pkg/log` for the usage.